instagram/src/App.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import ReactDom, { render } from 'react-dom';
+import 'bootstrap/dist/css/bootstrap.css';
+import { Button } from 'react-bootstrap';
+
```

Comment:
I like how you separate your dependancies from your components that you're importing in.